packages/rmb_direct_client/lib/client.ts

```diff
@@ -103,16 +104,17 @@ class Client {
       const twinId = await this.tfclient.twins.getTwinIdByAccountId({ accountId: this.signer.address });
       this.twin = await this.tfclient.twins.get({ id: twinId });
       if (!twinId) {
-        throw new Error({ message: `Couldn't find a user for the provided mnemonic on this network.` });
+        throw new ValidationError(`Couldn't find a user for the provided mnemonic on this network.`);
       }
     } catch (e) {
       console.log(e);
-      throw new Error({ message: `Couldn't find a user for the provided mnemonic on this network.` });
+      //TODO is this a valid error message?
+      throw new ConnectionError(`Couldn't find a user for the provided mnemonic on this network.`);
```

Comment:
I think this is a validation Error not connection. What do u think ?

Reply:
i think this one is the validation related 
https://github.com/threefoldtech/tfgrid-sdk-ts/pull/1433/files/d251636ea203fb9362bd10a0efc8b6deac7d9203#diff-3290bb9d40c106d5afd7fbca7c8c3d3bc60d1025e6c5b6634f0e244138fd215cR107
 
but in catch from my usage in playground i got this error many times while network is offline so i mentioned in line 111 to check this error message

Reply:
maybe we can change it to `Couldn't find a user for the provided mnemonic, check your connection`. or any msg with indication that the problem in the connection

Reply:
IMO, this try-and-catch block was added for having a more user-friendly error. But now tfchain should throw the correct error with its type, no?

Reply:
> IMO, this try-and-catch block was added for having a more user-friendly error. But now tfchain should throw the correct error with its type, no?

exactly but the question is what it the correct error in this case ?